csrc/scheduler/no_op.cpp

```diff
@@ -118,6 +118,11 @@ void NoOpScheduler::schedule(Fusion* fusion) {
     return;
   }
 
+  // Make sure we don't have global memory set on intermediate tensors from
```

Comment:
I think `scheduler_utils::clearMemorySpace` is run for all schedulers in their schedule function. e.g., https://github.com/NVIDIA/Fuser/blob/main/csrc/scheduler/pointwise.cpp#L532-L534

Reply:
That's right. Sorry if I missed anything -- are you requesting a change in the comment?

Reply:
No. I was just thinking that `scheduler_utils::clearMemorySpace` seems like a segmentation utility rather than a scheduler one.